### PR TITLE
Add Some Encounter Cards

### PR DESCRIPTION
**Pull request by adamgfraser**

### CODE REVIEW

File: pack/fhv/fhvc.json [hunk 1]
```diff
@@ -565,5 +658,48 @@
...
+    "code": "10727",
+    "encounter_code": "transfiguration",
+    "encounter_position": 3,
+    "faction_code": "mythos",
+    "illustrator": "Stephen Sommers",
+    "name": "Fungal Rot",
+    "pack_code": "fhvc",
+    "position": 227,
+    "quantity": 2,
+    "text": "<b>Revelation</b> – Attach Fungal Rot to an [[Item]] asset you control without Fungal Rot attached. If you cannot, Fungal Rot gains surge. \nTreat attached asset's text box as if it were blank (except for [[Traits]]). \n[action]: Test [combat] (3). If you succeed, discard Fungal Rot. Otherwise, take 1 damage.",
```
Comment (zzorba) [inline pack/fhv/fhvc.json]: Extra space before the \n here, and in other spots.

File: pack/fhv/fhvc.json [hunk 2]
```diff
@@ -261,6 +261,99 @@
...
+    "name": "Dawn of the First Day",
+    "pack_code": "fhvc",
+    "position": 182,
+    "quantity": 1,
+    "stage": 1,
+    "text": "Investigators cannot take damage/horror or be defeated. \n<b>Objective</b> - Befriend the locals and prepare for the first survey! <i>(Refer to the rules on page 6 for more information on preludes)</i>",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "You reconvene back the boarding house. Ms. Olmstead lays out a spread of warm buttered cornbread, roasted radishes the size of your fist, and a pile of grey oily eggs. Your stomach growls as you nibble on some of your field rations rather than partake. \nCheck the campaign log: \n<i>If Bertie was rescued,</i> (→R1) <i>(page 42)</i>, \nOtherwise, (→R2) <i>(page 42)</i>",
```
Comment (zzorba) [inline pack/fhv/fhvc.json]: I think Usually the R1 and R2 stuff is supposed to be bolded (not sure if these were formatted differently.

Also this looks like game text, should it be in the back_text instead?

File: pack/fhv/fhvc.json [hunk 3]
```diff
@@ -261,6 +261,99 @@
...
+    "position": 183,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Forced</b> - When you draw a weakness: cancel its revelation effect and discard it.",
+    "type_code": "agenda"
+  },
+  {
+    "back_name": "A Mistake",
+    "clues": null,
+    "back_text": "If you flipped this act while or after playing the Dawn of the Second Day prelude, read the following: <i>\nAs you walk through the village, a wave of dizziness washes over you and you stumble forward, putting your hand out to steady yourself. You cut your palm on the corner of a stack of mirrors and cry out in pain as thick, heavy raindrops begin to fall.</i> \nCodex 17 <i>(page 42).</i>",
```
Comment (zzorba) [inline pack/fhv/fhvc.json]: I think this is formatted like a blockquote.  If you look at the untamed wilds acts/agendas, there is special syntax for when acts/agendas have these 'inset lines' that alternate instructions + flavor.

File: pack/fhv/fhvc.json (hunk 2 above)
Comment (adamgfraser) [reply pack/fhv/fhvc.json]: Yes! Fixed now.

File: pack/fhv/fhvc.json (hunk 3 above)
Comment (adamgfraser) [reply pack/fhv/fhvc.json]: Done.

File: pack/fhv/fhvc.json (hunk 1 above)
Comment (adamgfraser) [reply pack/fhv/fhvc.json]: Fixed now.

File: pack/fhv/fhvc.json [hunk 4]
```diff
@@ -261,6 +261,99 @@
...
+    "position": 182,
+    "quantity": 1,
+    "stage": 1,
+    "text": "Investigators cannot take damage/horror or be defeated.\n<b>Objective</b> - Befriend the locals and prepare for the first survey! <i>(Refer to the rules on page 6 for more information on preludes)</i>",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "You reconvene back the boarding house. Ms. Olmstead lays out a spread of warm buttered cornbread, roasted radishes the size of your fist, and a pile of grey oily eggs. Your stomach growls as you nibble on some of your field rations rather than partake.",
+    "back_name": "Time to Get to Work!",
+    "back_text": "Check the campaign log:\n<i>If Bertie was rescued,</i> <b>(→R1)</b> <i>(page 42)</i>, \nOtherwise, <b>(→R2)</b> <i>(page 42)</i>",
```
Comment (zzorba) [inline pack/fhv/fhvc.json]: extra space before the \n here

File: pack/fhv/fhvc.json [hunk 5]
```diff
@@ -261,6 +261,99 @@
...
+    "position": 183,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Forced</b> - When you draw a weakness: cancel its revelation effect and discard it.",
+    "type_code": "agenda"
+  },
+  {
+    "back_name": "A Mistake",
+    "clues": null,
+    "back_text": "<b>If you flipped this act while or after playing the Dawn of the Second Day prelude, read the following:</b>\n<blockquote><i>As you walk through the village, a wave of dizziness washes over you and you stumble forward, putting your hand out to steady yourself. You cut your palm on the corner of a stack of mirrors and cry out in pain as thick, heavy raindrops begin to fall.</i></blockquote>\nCodex 17 <i>(page 42).</i>",
```
Comment (zzorba) [inline pack/fhv/fhvc.json]: Codex 17 should be `<b>([codex] 17)</b>`

File: pack/fhv/fhvc.json (hunk 4 above)
Comment (zzorba) [reply pack/fhv/fhvc.json]: I think this is missing a trailing Period on both lines as well, after the (page 42).

Comment (adamgfraser) [reply pack/fhv/fhvc.json]: Got it.

File: pack/fhv/fhvc.json (hunk 5 above)
Comment (adamgfraser) [reply pack/fhv/fhvc.json]: Ah I didn't realize we had something for that. Great!

File: pack/fhv/fhvc.json [hunk 6]
```diff
@@ -688,6 +718,93 @@
...
+    "text": "<b>Revelation</b> – Attach to the nearest non-[[Elite]] enemy. If you cannot, search the encounter deck and discard pile for a [[Mutated]] enemy and draw it.\n<b>Forced</b> – When attached enemy is defeated: Discard cards from the top of the encounter deck until an enemy is discarded and spawn it at this location.",
+    "traits": "Power. Colour.",
+    "type_code": "treachery"
+  },
+  {
+    "code": "10742",
+    "encounter_code": "mutations",
+    "encounter_position": 3,
+    "faction_code": "mythos",
+    "flavor": "\"It is not the strongest of the species that survives, not the most intelligent that survives. It is the one that is most adaptable to change\"\n-Charles Darwin",
```
Comment (zzorba) [inline pack/fhv/fhvc.json]: I think we need to put spaces around the - in the quotes, or else it might turn into a small bullet, like this: `\n - Charles`

Comment (adamgfraser) [reply pack/fhv/fhvc.json]: Done!

**Comment by zzorba** [comment]
Changes look good from my POV, but will let zooeyglass weigh in.

**Comment by adamgfraser** [comment]
Thanks for the reviews! 🙏

**Pull request merged**